test/algorithms/physical_design/graph_oriented_layout_design.cpp

```diff
@@ -209,14 +231,14 @@ TEST_CASE("Different cost objectives", "[graph-oriented-layout-design]")
 
     params.mode = graph_oriented_layout_design_params::effort_mode::HIGHEST_EFFORT;
 
-    // Array of cost objectives to iterate over
+    // array of cost objectives to iterate over
     const std::array<graph_oriented_layout_design_params::cost_objective, 5> cost_objectives = {
```

Comment:
As above:
```suggestion
    // array of cost objectives to iterate over
    const std::array cost_objectives = {
```
might work.